Model/Transaction.php

```diff
@@ -274,6 +282,30 @@ function buildCustomerExemption($order)
         return [];
     }
 
+    /**
+     * @param \Magento\Sales\Model\Order $order
+     * @return string
+     */
+    public function getProvider($order)
```

Comment:
@ryanreeves-taxjar This function should have a `protected` scope.

Reply:
I made this public when we were first looking at using a plugin around this function and forgot to change it back.